src/openrct2/localisation/Formatting.h

```diff
@@ -23,6 +23,48 @@ namespace OpenRCT2
 {
     using FormatToken = uint32_t;
 
+    class FmtString
+    {
+    private:
+        std::string_view _str;
+        std::string _strOwned;
```

Comment:
Don't use use TitleCase for member variables?

Reply:
This is lower camel case, the convention for private fields.